tests/ontopy_tests/test_graph2.py

```diff
@@ -1,14 +1,43 @@
 from typing import TYPE_CHECKING
+import owlready2
+import pytest
+from ontopy.graph import OntoGraph
 
 if TYPE_CHECKING:
     from pathlib import Path
 
     from ontopy.ontology import Ontology
 
 
-def test_graph2(emmo: "Ontology", tmpdir: "Path") -> None:
-    from ontopy.graph import OntoGraph
+@pytest.mark.filterwarnings(
+    "ignore:Style not defined for relation hasSpecialRelation."
+)  # currently pytest is set to accept warnings, but this might change in the future
+def test_graph(testonto: "Ontology", tmpdir: "Path") -> None:
+    with testonto:
+        # Add a relation that does not have a default style in OntoGraph
+        class hasSpecialRelation(owlready2.ObjectProperty):
+            "New special relation"
+            domain = list(testonto.classes())
+            range = list(testonto.classes())
 
+        class NewSpecialClass(owlready2.Thing):
+            "New class"
+
+    testonto.sync_attributes()
+
+    testonto.TestClass.hasSpecialRelation.append(testonto.NewSpecialClass)
+    graph = OntoGraph(
+        testonto,
+        testonto.TestClass,
+        relations="all",
+        addnodes=True,
+        edgelabels=None,
+    )
+    graph.add_legend()
+    graph.save(tmpdir / "testonto.png")
+
+
+def test_emmo_graphs(emmo: "Ontology", tmpdir: "Path") -> None:
     graph = OntoGraph(
```

Comment:
While you didn't change or update this test explicitly, I see this is also missing a doc-string similar to the I suggested above.